pkgs/development/libraries/ilmbase/cross.patch

```diff
@@ -0,0 +1,35 @@
+From: Helmut Grohne <>
```

Comment:
We had quite a few cross-build patches lately.
I think I should open a ticket, where reference the PRs for upstreaming.

Reply:
This is from an upstream issue that hasn't received any attention.

Reply:
Yeah. I could make it a proper pull request though. They maybe don't read all issues.